core/templates/pages/exploration-editor-page/translation-tab/services/translation-status.service.ts

```diff
@@ -187,10 +190,34 @@ export class TranslationStatusService implements OnInit {
         let stateNeedsUpdate = false;
         let noTranslationCount = 0;
         let noVoiceoverCount = 0;
-        let recordedVoiceovers =
-          this.explorationStatesService.getRecordedVoiceoversMemento(stateName);
+        let nonEmptyContent =
+          this.explorationStatesService.getNonEmptyRecordedVoiceoversMemento(
+            stateName
+          );
+
+        if (
```

Comment:
What happens when this condition doesn't hold? I am a bit confused about why it depends on whether the voiceover mode is active; shouldn't this just rely on what's in the exploration and what voiceovers are present?